trident/ray_generator.py

```diff
@@ -640,32 +640,42 @@ def _determine_fields_from_ions(ds, ion_list, fields, ftype, sampling_type):
         metallicity_field = "%s_metallicity" % atom
         field = "%s_p%d_number_density" % (atom, ion_state-1)
 
+        if ion_state == 1:
+            alias_field = "%s_number_density" % atom
+        else:
+            alias_field = field
+
         # This is ugly, but I couldn't find a way around it to hit
         # all 6 cases of when fields were present or not and particle
         # type or not.
         if (ftype, field) not in ds.derived_field_list:
```

Comment:
Changed this block of code back to how it originally was so the only change now has been renaming the field/alias_field for the ion_state==1 case.

Reply:
Awesome!  Thanks!